M31_Example/datamodel.py

```diff
@@ -0,0 +1,354 @@
+# (either CW or CCW).
+ast_coord_boundary = None
+
+
+# -------------------------------------------
+# Noise Model Artificial Star Test Parameters
+# -------------------------------------------
+
+# astfile : string
+#   pathname of the AST files (single camera ASTs)
+astfile = "M31-B09-EAST_chunk.gst.fake.fits"
+#astfile = "M31-B09-EAST_chunk.gst.fake_cut.fits"
+
+# ast_colnames : list of strings
+#   names of columns for filters in the AST catalog (AC)
+ast_colnames = np.array(basefilters)
+
+# noisefile : string
+#   create a name for the noise model
+noisefile = project + "/" + project + "_noisemodel.hd5"
+
+# absflux calibration covariance matrix for HST specific filters (AC)
+absflux_a_matrix = absflux_covmat.hst_frac_matrix(filters)
+
+# -------------------------------------------
+# Grid
+# -------------------------------------------
+
+# n_subgrid : integer
+#     Number of sub-grids to use (1 means no subgrids).  These are
+#     useful when the physics model grid is too large to read into
+#     memory.
+n_subgrid = 1
+
+################
+
+# Distance/Velocity
+
+# Distances: distance to the galaxy [min, max, step] or [fixed number]
+distances = [24.79]#[24.29, 25.29, 0.25] #number was originally 24.79
+#used 2013AJ....146...86T as a reference
+
+# Distance unit (any length or units.mag)
+distance_unit = units.mag
+
+# velocity of galaxy
+# velocity should be heliocentric
+velocity = -179 * units.km / units.s
+
+################
+
+# Stellar grid definition
+
+# log10(Age) -- [min,max,step] to generate the isochrones in years
+#   example [6.0, 10.13, 1.0]
+logt = [6.0, 10.13, 1.0]
+age_prior_model = {'name': 'flat'}
+
+# note: Mass is not sampled, instead the isochrone supplied
+#       mass spacing is used instead
+mass_prior_model = {'name': "kroupa"}
+
+# Metallicity : list of floats
+#   Here: Z == Z_initial, NOT Z(t) surface abundance
+#   PARSECv1.2S accepts values 1.e-4 < Z < 0.06
+#   example z = [0.03, 0.019, 0.008, 0.004]
+#   can they be set as [min, max, step]?
+z = ([0.03, 0.019, 0.008, 0.004])
+met_prior_model = {'name': 'flat'}
+#    10 ** np.array([-2.1, -1.8, -1.5, -1.2, -0.9, -0.6, -0.3, 0.0, 0.3]) * 0.0152
+#).tolist()
+
+# Isochrone Model Grid
+#   Current Choices: Padova or MIST
+#   PadovaWeb() -- `modeltype` param for iso sets from ezpadova
+#      (choices: parsec12s_r14, parsec12s, 2010, 2008, 2002)
+#   MISTWeb() -- `rotation` param (choices: vvcrit0.0=default, vvcrit0.4)
+#
+# Default: PARSEC+CALIBRI
+oiso = isochrone.PadovaWeb()
+# Alternative: PARSEC1.2S -- old grid parameters
+# oiso = isochrone.PadovaWeb(modeltype='parsec12s', filterPMS=True)
+# Alternative: MIST -- v1, no rotation
+# oiso = isochrone.MISTWeb()
+
+# Stellar Atmospheres library definition
+osl = stellib.Tlusty() + stellib.Kurucz()
+
+################
+
+# Dust extinction grid definition
+extLaw = extinction.Generalized_RvFALaw(
+    ALaw=extinction.Generalized_DustExt(curve="F20"),
```

Comment:
F20 needs to be changed to F19.  Newer version of `dust_extinction` has the F19 name for this model (paper was published faster than expected).